PowerPointLabs/PowerPointLabs/CropLab/CropLabUIControl.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Windows;
+
+namespace PowerPointLabs.CropLab
+{
+    internal class CropLabUIControl
```

Comment:
1. make it implement an interface, e.g. an interface contains method `ShowErrorMessageBox`
2. instead of making it singleton, create a factory static method, like `GetCropLabMessageService` to return the UIControl, and another static method `GetCropLabDummyMessageService` to return dummy control for your UT